--- /opt/anaconda3/envs/jupyter/lib/python3.9/site-packages/notebook/static/components/codemirror/mode/meta.js	2021-03-22 13:21:20.000000000 +0000
+++ meta.js	2021-06-02 12:25:41.366504172 +0000
@@ -114,6 +114,7 @@
     {name: "Puppet", mime: "text/x-puppet", mode: "puppet", ext: ["pp"]},
     {name: "Q", mime: "text/x-q", mode: "q", ext: ["q"]},
     {name: "R", mime: "text/x-rsrc", mode: "r", ext: ["r", "R"], alias: ["rscript"]},
+    {name: "Raku", mime: "text/x-raku", mode: "raku", ext: ["raku", "rakutest", "rakumod", "nqp", "p6", "pl6", "pm6"], alias: ["perl6"]},
     {name: "reStructuredText", mime: "text/x-rst", mode: "rst", ext: ["rst"], alias: ["rst"]},
     {name: "RPM Changes", mime: "text/x-rpm-changes", mode: "rpm"},
     {name: "RPM Spec", mime: "text/x-rpm-spec", mode: "rpm", ext: ["spec"]},
